src/libs/ReportUtils.ts

```diff
@@ -1257,6 +1257,23 @@ function isMoneyRequestReport(reportOrID: OnyxEntry<Report> | EmptyObject | stri
     return isIOUReport(report) || isExpenseReport(report);
 }
 
+/**
+ * Checks if a report has only one transaction associated with it
+ */
+function isOneTransactionReport(reportID: string): boolean {
+    const reportActions = ReportActionsUtils.getAllReportActions(reportID);
+    return ReportActionsUtils.getOneTransactionThreadReportID(reportActions) !== '0';
```

Comment:
As mentioned above it would be nice to look for null here instead of specific `'0'` string here